rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction-extractfrompdf.adoc

```diff
@@ -0,0 +1,64 @@
+# Finding Information in a PDF
+// number the headings to correspond to the numbered list of steps
+:sectnums:
+
+First, you need to find the order ID in the PDF.
+
+As you can see in xref:automation-usecase-legacysystemdataextraction.adoc#confirmationletter-image[the image of the order confirmation letter], the order ID is always listed in a table next to the entry 'Order ID'. 
+// Do we need an image of that part of the letter here?
+You can use this information to implement the workflow that contains the following steps:
+
+. Copy the content of the order confirmation to a text file.
+. Read the text file line by line to find the one containing the order ID.
+. Check if the order ID is found
+. Cut the order ID out from the text and store it in an activity parameter.
+
+image::usecase-legacysystem-extractformpdf-workflow-overview.png["The Workflow contains the steps in form of Action Steps. Some of the Action Steps are nested."]
+
+## Copying Text From a PDF to a Text File
+To open the PDF, you use *Read PDF file*. Then you use the returned `ReadText` as input for *Write to Text File* to save the content as text file on your computer.
+
+image::usecase-legacysystem-extractformpdf-workflow-pdftotxt.png["An out-filled pin symbol next to the input field shows that the variable is mapped to the text extracted from the PDF in a previous Action Step."]
+
+The following image shows the result of storing the text content. Each line of the PDF has a corresponding line in the text file. Images are omitted.
+
+// no alt text because description of image is contained in paragraph
+image::extractorderid-workflow-resultpdftotxt.png['', 75%]
+
+## Looping Over All Lines of a Text File
+
+To loop over all lines of the text file, you first need to count the number of lines by reading the entire file using the Action Step *Read from Text File*. You use the returned `line count` as the maximum value for the iteration counter. The counter doubles as the number of the line to read with *Read from Text File* in each iteration. 
```

Comment:
"The counter doubles" might be confusing. Do you mean that it doubles the count or that it also functions as the number of the line...?